terms.html

```diff
@@ -1,254 +1,225 @@
-<a href="https://en.wikipedia.org/wiki/Bitcoin">Bitcoin</a>,
-the term "blockchain" is sometimes used to refer specifically to the Bitcoin
+A specific type of <a>distributed ledger technology</a> (DLT) that stores ledger
+entries in blocks of transactions, which are grouped together and hashed into a
+cryptographic chain. Because this type of <a>DLT</a> was introduced by
+<a href="https://en.wikipedia.org/wiki/Bitcoin">Bitcoin</a>, the term
+<em>blockchain</em> is sometimes used to refer specifically to the Bitcoin
 ledger.
   </dd>
 
-
-  <dt><dfn data-lt="DID|DIDs">decentralized identifier</dfn> (DID)</dt>
+  <dt><dfn data-lt="decentralized identifiers|DID|DIDs">decentralized identifier</dfn> (DID)</dt>
 
   <dd>
-A globally unique identifier that does
-not require a centralized registration authority because it is
-registered with <a>distributed ledger technology</a> (DLT) or other form of
-decentralized network. The generic format of a DID is defined in this
-specification. A specific <a>DID scheme</a> is defined in a
+A globally unique identifier that does not require a centralized registration
+authority because it is registered with <a>distributed ledger technology</a>
+(DLT) or other form of decentralized network. The generic format of a DID is
+defined in this specification. A specific <a>DID scheme</a> is defined in a
 <a>DID method</a> specification.
   </dd>
 
-
-  <dt><dfn data-lt="">decentralized identity management</dfn></dt>
+  <dt><dfn>decentralized identity management</dfn></dt>
 
   <dd>
 <a href="https://en.wikipedia.org/wiki/Identity_management">Identity
-management</a> based on <a>decentralized identifiers</a>.
-Decentralized identity management extends the identifier creation authority
-beyond the traditional roots of trust
-required by <a href="https://en.wikipedia.org/wiki/X.500">X.500
-directory services</a>, the <a href=
-  "https://en.wikipedia.org/wiki/Domain_Name_System">Domain Name
-System</a>, and most national ID systems.
+management</a> based on <a>decentralized identifiers</a>. Decentralized identity
+management extends the identifier creation authority beyond the traditional
+roots of trust required by
+<a href="https://en.wikipedia.org/wiki/X.500">X.500 directory services</a>,
+the <a href="https://en.wikipedia.org/wiki/Domain_Name_System">Domain Name System</a>,
+and most national identification systems.
   </dd>
 
-
-  <dt><dfn data-lt="">decentralized public key infrastructure</dfn> (DPKI)</dt>
+  <dt><dfn>decentralized public key infrastructure</dfn> (DPKI)</dt>
 
   <dd>
-Public key infrastructure based on <a>decentralized identifiers</a>
-and identity records (e.g., <a>DID documents</a>) containing verifiable
+Public key infrastructure based on <a>decentralized identifiers</a> and identity
+records (for example, <a>DID documents</a>) containing verifiable
 <a>public key descriptions</a>.
   </dd>
 
-
-  <dt><dfn data-lt="">delegate</dfn></dt>
+  <dt><dfn data-lt="delegates">delegate</dfn></dt>
```

Comment:
There's a flag in respec we can enable (`pluralize: true`) to do this automatically.. will set that at some point.

Reply:
Shall I remove the plural forms then?

Reply:
That flag is not currently enabled in the respec config, and they don't hurt, so no. Just noting for future reference.